notebooks/compendium/teleportation/index.ipynb

```diff
@@ -0,0 +1,464 @@
+{
```

Comment:
<p>I feel this tone in the first two paragraphs is too conversational. So far I've been aiming for "Serious / Casual / Respectful / Matter-of-fact" in the <a href="https://www.nngroup.com/videos/tone-of-voice-dimensions/" target="_blank">four dimensions of tone</a>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/textbook/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>